governance/votes/Executive vote - June 7, 2023.md

```diff
@@ -0,0 +1,215 @@
+- [Liquidation Ratio (`mat`)](https://manual.makerdao.com/parameter-index/vault-risk/param-liquidation-ratio): **100%**.
+- Oracle Price: **2.5 million DAI**.
+- Debt write-off timelock (`tau`): **0**.
+- Liquidation Process: Will use the established RWA collateral contracts. See [MIP21](https://mips.makerdao.com/mips/details/MIP21) for details.
+- [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement): QmdbPyQLDdGQhKGXBgod7TbQmrUJ7tiN9aX1zSL7bmtkTN - [view here](https://gateway.pinata.cloud/ipfs/QmdbPyQLDdGQhKGXBgod7TbQmrUJ7tiN9aX1zSL7bmtkTN).
+
+#### Relevant Smart Contract Information
+
+- **MIP21_LIQUIDATION_ORACLE:** `0x88f88Bb9E66241B73B84f3A6E197FbBa487b1E30`. 
+- **RWA_TOKEN_FAB:** `0x2B3a4c18705e99bC29b22222dA7E10b643658552`.
+- **SYMBOL:** `RWA015`.
+- **NAME:** `RWA-015`.
+- **ILK:** `RWA015-A`.
+- **RWA014:** `TBD`
+- **MCD_JOIN_RWA015_A:** `TBD`.
+- **RWA015_A_URN:** `TBD`.
+- **RWA015_A_JAR:** `TBD`.
+- **RWA015_A_OUTPUT_CONDUIT:** `TBD`.
+- **RWA015_A_INPUT_CONDUIT_URN:** `TBD`.
+- **RWA015_A_INPUT_CONDUIT_JAR:** `TBD`.
+- **RWA015_A_OPERATOR:** `TBD`.
+- **ETH_Address_A (USDC Wallet In):** `0x65729807485F6f7695AF863d97D62140B7d69d83`.
+
+#### Additional Information
+
+Please review the following forum threads containing information about RWA015 (BlockTower Andromeda) to inform your position before voting:
+- [Initial Collateral Application](https://forum.makerdao.com/t/mip90-liquid-aaa-structured-credit-money-market-fund/18428).
+- [Steakhouse Financial Legal & Risk Assessment](https://forum.makerdao.com/t/project-andromeda-risk-legal-assessment/20969).
+- [dewiz Technical Assessment](https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974).
+
+### BlockTower RWA Vault [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmPMrvfV) the following changes to the BlockTower RWA Vault Debt Ceiling parameters will be made, if this executive proposal passes:
+- Decrease the RWA010-A Debt Ceiling by 20 million DAI from 20 million DAI to **0**.
+- Decrease the RWA011-A Debt Ceiling by 30 million DAI from 30 million DAI to **0**.
+- Increase the RWA012-A Debt Ceiling by 50 million DAI from 30 million DAI to **80 million DAI**.
+
+To facilitate these changes, the [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) for RWA010-A, RWA011-A, RWA012-A, and RWA013-A will be updated to **QmY382BPa5UQfmpTfi6KhjqQHtqq1fFFg2owBfsD2LKmYU**, the agreement details may be [viewed here](https://gateway.pinata.cloud/ipfs/QmY382BPa5UQfmpTfi6KhjqQHtqq1fFFg2owBfsD2LKmYU).
+
+### Stability Scope Defined Parameter Changes
+
+As per the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104), the following parameter changes will be made, if this executive proposal passes:
```

Comment:
Is it worth mentioning that all these changes are a result of the modified base rate, and potentially linking more directly to the relevant parts of the Stability Scope?